api/src/main/java/org/openmrs/module/ModuleFactory.java

```diff
@@ -128,7 +128,7 @@ public static Module loadModule(File moduleFile, Boolean replaceIfExists) throws
 	public static Module loadModule(Module module, Boolean replaceIfExists) throws ModuleException {
 		
 		if (log.isDebugEnabled()) {
-			log.debug("Adding module " + module.getName() + " to the module queue");
+			log.debug("Adding module to the module queue {}", module.getName());
```

Comment:
You should aim to keep the log messages before & after the change identical. Here you changed `"Adding module yyy to the module queue"` to `"Adding module to the module queue yyy"`

Reply:
@gracebish kindly fix your ticket id and also give a description of what you worked on.